woocommerce/woocommerce
# Fix styles for modals

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR addresses style issues that have affected several modals.

![Screenshot 2023-06-15 at 11 55 22](https://github.com/woocommerce/woocommerce/assets/1314156/b9778542-c19c-4d8c-80fb-66bd302c88bd)

Closes #38748.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Set up a brand new WC store.
2. Go to WooCommerce > Settings > Advanced > Features and enable the product blocks editor.
3. Navigate to the OBW and fill out the `Store details` form, then click `Continue`. 
4. Verify that the `Build a better WooCommerce` modal is displayed correctly.

![Screenshot 2023-06-16 at 14 24 07](https://github.com/woocommerce/woocommerce/assets/1314156/ebdaad98-33f4-4acc-8596-0ee692a90093)
 
5. Go to Products > Add New > Shipping and select `New shipping class`.
6. Ensure that the modal appears correctly.
7. Click the dropdown menu next to the `Add` button and select `Share feedback`.

![Screenshot 2023-06-16 at 14 26 10](https://github.com/woocommerce/woocommerce/assets/1314156/7de556fe-d679-4ca0-8258-d812c9ab7524)

8. Verify that the CES (Customer Effort Score) modal is properly displayed.
